packages/uilib/src/lib/plugins/communication-explorer/sidebar/ied-accordion/ied-accordion.svelte

```diff
@@ -18,19 +19,42 @@
     $: relations = getConnectedIEDsByLabel(rootNode, IEDSelection.label)
     $: relationsByServiceType = groupRelationsByServiceType(relations)
     $: serviceTypes = Array.from(relationsByServiceType.entries())
+    $: details = getIEDDetails(rootNode, IEDSelection.label)
 
     const serviceTypeColor: { [key in MessageType | "Unknown"]: string } = {
     	GOOSE:         "--color-message-goose",
     	MMS:           "--color-message-mms",
     	SampledValues: "--color-message-sampledvalues",
     	Unknown:       "--color-message-unknown",
     }
+
+    let detailsCollapsed = true;
 </script>
 
 <div class="ied">
     <IED label={IEDSelection.label} isSelected={true} isSelectable={false} />
 </div>
-<div class="accordions">
+{#if details.length == 3}
+    <div class={detailsCollapsed ? 'details_collapsed' : ''}>
+    {#if details[0] != ""}
```

Comment:
Do you want to test the [falsiness](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) of the value or that it is an empty string ? 
We should enforce the use of `===` strict equality.